lib/views/help/glossary.html.erb

```diff
@@ -32,13 +32,16 @@
 <dt id="absolute_exemption">Absolute exemption</dt>
 <dd>An absolute exemption is an <a href="#exemption">exemption</a> that is not subject to a <a href="#public_interest_test">public interest test</a>.</dd>
 <dt id="alaveteli">Alaveteli</dt>
-<dd>An open-source platform for making public <a href="#freedom_of_information">freedom of information</a> requests to <a href="#public_authorities">public authorities</a>. <a href="#whatdotheyknow">WhatDoTheyKnow</a> is powered by Alaveteli.</dd>
+<dd>An open-source platform for making public <a href="#freedom_of_information">freedom of information</a> requests to <a href="#public_authority">public authorities</a>. <a href="#whatdotheyknow">WhatDoTheyKnow</a> is powered by Alaveteli.</dd>
 <dt id="">Anonymisation</dt>
 <dd>The process of removing names, National Insurance numbers and other identifiers from a dataset prior to release so that none of the information can be linked to a specific person. Anonymised data may be useful to researchers.</dd>
 <dt id="api">API</dt>
 <dd>API stands for Application Programming Interface. An API is a piece of software that allows two computer programmes to exchange information. See also: <a href="https://www.whatdotheyknow.com/help/api">https://www.whatdotheyknow.com/help/api</a> </dd>
 <dt id="">Applicant blind</dt>
-<dd>The principle that the information released in response to an <a href="#eir">EIR</a> or <a href="#foi">FOI</a> request should be the same no matter who the <a href="#requester">requester</a> is. In practice, <a href="#public_authorities">public authorities</a> can take the identity of the requester into account in certain situations e.g. when applying the <a href="#cost_limit">cost limit</a>.</dd>
+<dd>The principle that the information released in response to an <a href="#eir">EIR</a> or <a href="#foi">FOI</a> request should be the same no matter who the <a href="#requester">requester</a> is. In practice, <a href="#
+ 
+ 
+ orities">public authorities</a> can take the identity of the requester into account in certain situations e.g. when applying the <a href="#cost_limit">cost limit</a>.</dd>
```

Comment:
Doesn't look like this should have newlines breaking the href?

Reply:
Also, think it needs the same change from `#public_authorities` to `#public_authority`.